orchagent/aclorch.cpp

```diff
@@ -935,7 +951,10 @@ bool AclTable::bind(sai_object_id_t portOid)
     assert(ports.find(portOid) != ports.end());
 
     sai_object_id_t group_member_oid;
-    if (!gPortsOrch->bindAclTable(portOid, m_oid, group_member_oid))
+
+    bool ingress = (stage == ACL_STAGE_INGRESS) ? true : false;
+
+    if (!gPortsOrch->bindAclTable(portOid, m_oid, group_member_oid, ingress))
```

Comment:
>ingress [](start = 68, length = 7)

It's better to pass original acl_stage_type_t

Reply:
I would prefer to pass boolean in this function. bindAclTable is a member function of PortsOrch. It doesn't have to know the detail of the existence acl_stage_stype_t which is more specific for aclOrch. Instead, a boolean to indicate ingress/egress should be enough for this interface.